binomial_cis/binomial_helper.py

```diff
@@ -31,10 +31,25 @@ def binom_pmf(k, n, p):
     Returns
     pmf: binomial pmf evaluated at k, n, p
     """
-    pmf = binom_coeff(n, k) * p**k * (1-p)**(n-k)
-    
-    # safeguard against floating point errors
-    pmf = min(1, max(0, pmf)) # unable to use np.clip()
+    if p == 1:
+        if k == n:
+            pmf = 1
+        else:
+            pmf = 0
+    elif p == 0:
+        if k == 0:
+            pmf = 1
+        else:
+            pmf = 0
+    else:
+        # following code from https://stackoverflow.com/a/45869209
+        log_binom_coeff = 0
+        for i in range(1, int(k) + 1): # should loop from i = 1, ..., k
```

Comment:
Should this be `range(1, int(min(k, n- k)) + 1)` to be consistent with the current implementation of `binom_coeff`?
```suggestion
        for i in range(1, int(min(k, n - k)) + 1): # should loop from i = 1, ..., k
```

Reply:
cc: @Joe-Vincent

Reply:
Yes, let's use the same upper limit as used in `binom_coeff`.  
It can save on unnecessary computation.